v2/elasticsearch-common/src/main/java/com/google/cloud/teleport/v2/elasticsearch/options/ElasticsearchWriteOptions.java

```diff
@@ -29,17 +29,35 @@
 
   void setTargetNodeAddresses(String targetNodeAddresses);
 
-  @Description("The index toward which the requests will be issued, ex: my-index")
+  @Description("The document type toward which the requests will be issued, ex: my-document-type")
+  @Default.String("_doc")
+  String getWriteDocumentType();
+
+  void setWriteDocumentType(String writeDocumentType);
+
+  @Description("Write username for elasticsearch endpoint")
   @Validation.Required
-  String getWriteIndex();
+  String getWriteElasticsearchUsername();
 
-  void setWriteIndex(String writeIndex);
+  void setWriteElasticsearchUsername(String writeElasticsearchUsername);
 
-  @Description("The document type toward which the requests will be issued, ex: my-document-type")
+  @Description("Write password for elasticsearch endpoint")
   @Validation.Required
-  String getWriteDocumentType();
+  String getWriteElasticsearchPassword();
 
-  void setWriteDocumentType(String writeDocumentType);
+  void setWriteElasticsearchPassword(String writeElasticsearchPassword);
+
+  @Description("Write Dataset used to build index in format: logs-gcp.{Dataset}-{Namespace}")
```

Comment:
The type of logs sent via Pub/Sub for which we have out of the box dashboard. Known log types values are audit, vpcflow, and firewall. If no known log type is detected, we default to pubsub